frontend/src/components/maps/leaflet/PointClusterer.tsx

```diff
@@ -302,7 +312,13 @@ export const PointClusterer: React.FC<PointClustererProps> = ({
               icon={getMarkerIcon(cluster)}
               onclick={() => {
                 onMarkerClick(); //open information slideout
-                fetchProperty(cluster.properties.propertyTypeId, cluster.properties.id);
+                if (keycloak.canUserViewProperty(cluster.properties as IProperty)) {
```

Comment:
if the user can view the property, use API data, otherwise convert and use the map data